gammapy/scripts/catalog_query.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ..utils.scripts import get_parser
+
+__all__ = ['catalog_query']
+
+def main(args=None):
+    parser = get_parser(catalog_query)
+    parser.add_argument('--catalog',
+                        help='Catalog for the source e.g. 3FGL')
+    parser.add_argument('--source',
+                        help='Source name e.g. J0349.9-2102')
+    parser.add_argument('--querytype',
+                        help='The query type: info, lightcurve, or spectrum')
+    catalog_query(**vars(args))
+
+def catalog_query(catalog, source, querytype):
+    """Query a catalog
+
+    TODO: explain.
+    """
+    from gammapy.datasets import fermi
+
+    if (catalog=='3FGL'):
+        catalog_object = fermi.Fermi3FGLObject(source)
+    else:
+        raise NotImplementedError
+
+    if querytype == 'info':
+        print(source)
+        print ("RA (J2000) " + str(catalog_object.ra))
```

Comment:
Can you make a `Fermi3FGLObject.info` method that returns this string?
Then just call it here and print it.

This is more re-usable, allowing to print the info also e.g. in interactive use. The command line tools should just be thin wrappers around existing functions / methods, that do argument parsing and handling, calling existing functions / methods.